dist/exec-child.js

```diff
@@ -0,0 +1,39 @@
+if (require.main !== module) {
```

Comment:
This is the compiled code folder, required for the GitHub Action. I can't change the output or it will be changed back. I'll add a README in the folder to explain.